### ISSUE TITLE
How to load custom settings from selenium?

### ISSUE THREAD

**Issue by jamb0ss** [kkapsner/CanvasBlocker#359]
Is there any simple way to load custom settings (`CanvasBlocker-settings.json`) with a new Firefox profile from Selenium (no user interaction, I can only run some pre-defined JS code in the context of `moz-extensions://canvas-blocker-id` page)? It seems that you don't use `browser.storage` API to store this data that I can simply interact with from my selenium script, so I just can't figure out how to load my settings for CanvasBlocker with an absolutely new and fresh Firefox profile? It won't work via CanvasBlocker settings page (`Load` button), because it's not an `input` element and therefor Selenium can't send there a full path to my settings file. And if I click this button, then I can't access the system dialog from Selenium again.

**Comment by kkapsner** [comment]
If you can run code in the context of CB you should be able to run similar code to https://github.com/kkapsner/CanvasBlocker/blob/master/options/options.js#L97

**Comment by kkapsner** [comment]
The settings are stored via browser.storage.local.set(). So you can either set them there directly or you can use the settings-API from CB.

Example:
```browser.storage.local.set({blockMode: "ask"});```
```const settings = require("./lib/settings");
settings.blockMode = "ask";```

But if you do not use the CB API you have to make sure you only provide valid settings. The API will check if the values are correct.

**Comment by jamb0ss** [comment]
I got it, when I checked the `browser.storage` there were no settings, that why I decided, that you use another way to store settings. My fault was that I used the brand new installation of your addon, therefor all settings were default, and `browser.storage` had no corresponding keys. When I explicitly load my custom settings, the `browser.storage` updated and I see all what I was looking for there. Thanks!
